src/Page/CaseStudy.elm

```diff
@@ -59,3 +75,91 @@ viewMaybeQuote maybeQuote =
 
         Nothing ->
             text ""
+
+
+
+-- Styles
```

Comment:
Is it worth moving any of these styles into Theme for use by other pages? Or do you think all pages are significantly different enough that they will need own styles.

Reply:
I don't think the page sections have a similar enough layout to share too much else at the moment.